yaptide/celery/utils/pymc.py

```diff
@@ -81,7 +82,7 @@ def command_to_run_fluka(dir_path: Path, task_id: str) -> list[str]:
         input_path=dir_path,  # skipcq: PYL-W0612 # usefull
         simulator_type=SimulatorType.fluka,
         simulator_exec_path=None,  # useless, we guess from PATH
-        cmdline_opts="")  # useless, not useful for fluka
+        cmdline_opts="-M 1")  # useless, not useful for fluka
```

Comment:
why this option ?

Reply:
I will add an appropriate comment.

About this option:
Fluka by default executes  5 simulations one after another.
From fluka -h (help) sommand
```
  This operation is repeated from the cycle after the one indicated
       on the -N option (default 0) to the one indicated on the -M
       option (default 5).
...

       -N<last_cycle>   number of the previous run cycle number
       -M<final_cycle>  final run number
```

Reply:
Add a comment please. Usually we run multiple cycles when working in command line. That gives us error estimation when running on single CPU